tokio/src/runtime/mod.rs

```diff
@@ -173,10 +173,11 @@
 //! [`Builder::enable_all`]: crate::runtime::Builder::enable_all
 
 // At the top due to macros
-#[cfg(test)]
-#[cfg(not(target_arch = "wasm32"))]
-#[macro_use]
-mod tests;
+cfg_not_wasi! {
```

Comment:
#[cfg(not(target_arch = "wasm32"))] is not equivalent to cfg_not_wasi!

Reply:
Resolved, I put back the existing !wasm check